docker-for-aws/iam-permissions.md

```diff
@@ -309,6 +309,16 @@ follow the link for more information.
             "Resource": [
                 "*"
             ]
+        },
+        {
+            "Sid": "Stmt1487169681000",
+            "Effect": "Allow",
+            "Action": [
+                "elasticfilesystem:*"
```

Comment:
Maybe we should also add `ec2:ModifyNetworkInterfaceAttribute` as prescribed in AmazonElasticFileSystemFullAccess in IAM console.